metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_v2.py

```diff
@@ -493,6 +490,24 @@ def query(query):
 
         return _report
 
+    def _init_schema_resolver(self) -> SchemaResolver:
+        if not self.config.include_technical_schema and self.config.parse_view_ddl:
+            if self.ctx.graph:
+                return self.ctx.graph.initialize_schema_resolver_from_datahub(
```

Comment:
instead of initialize_schema_resolver_from_datahub returning `Tuple[SchemaResolver, set[urns]]`, can we change it so that it only returns SchemaResolver, and in turn SchemaResolver has a method to get a set of loaded urns?

Reply:
makes sense. What would we call this method in SchemaResolver ? `get_urns` ?